BUILD/targets/zephyr/Makefile.app

```diff
@@ -27,7 +27,7 @@ SOLETTA_CFLAGS += -Wno-missing-include-dirs
 
 export SOLETTA_CFLAGS SOLETTA_LDFLAGS
 
-ZEPHYRINCLUDE += -I$(SOLETTA_INCLUDE_PATH)
+ZEPHYRINCLUDE += -I$(SOLETTA_INCLUDE_PATH) -DSOL_LOG_LEVEL=\"5\"
```

Comment:
I will parametrize and have a default value, then. I want this to provide logs to report bugs.